hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/upgrade/QuotaRepairUpgradeAction.java

```diff
@@ -35,7 +35,8 @@ public class QuotaRepairUpgradeAction implements OmUpgradeAction {
   @Override
   public void execute(OzoneManager arg) throws Exception {
     boolean enabled = arg.getConfiguration().getBoolean(
-        OMConfigKeys.OZONE_OM_UPGRADE_QUOTA_RECALCULATE_ENABLE, false);
```

Comment:
In addition, the default value should be consistent with ozone-default.xml

Reply:
yes, we can set default as true. I was thinking if not configured, then go with false, but this can be changed.

Reply:
> yes, we can set default as true. I was thinking if not configured, then go with false, but this can be changed.

@sumitagrawl  `true` in ozone-default.xml will take effect if ozone-site.xml not configured.  So, I just set the value of OZONE_OM_UPGRADE_QUOTA_RECALCULATE_ENABLE to true to avoid misleading.